src/main/java/seedu/address/model/person/Notes.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.model.person;
+import static java.util.Objects.requireNonNull;
+/**
+ * Represents a Patient's notes in WardWatch.
+ * Guarantees: immutable; is always valid
+ */
+public class Notes {
+    public final String notes;
```

Comment:
Very minor nit, can consider changing the field name to value, to match that of other Field classes.